recipes/cuda-python/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "cuda-python" %}
+{% set version = "11.4.0" %}
+{% set major_version = version.split(".")[0] %}
+{% set minor_version = version.split(".")[1] %}
+{% set patch_version = version.split(".")[2] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e1c1bd49919c8646b7d84e3e63f5a0a971b64a69a9c986194d66f17cd26ee006
+
+build:
+  number: 0
+  skip: true  # [py<37 or osx]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
```

Comment:
You need a `recipe/cuda-python/conda-forge.yml` with `osx_version: linux_64: cos7` added as well.

Reply:
Thanks for the reminder Isuru. Just to confirm that is `os_version` right?

Reply:
yes

Reply:
Updated.